### PR TITLE
Test travis

### PR DESCRIPTION
**Pull request by prabhjyotsingh**

### CODE REVIEW

**Comment by prabhjyotsingh** [comment]
asdfasdf 11

File: .travis.yml [hunk 1]
```diff
@@ -21,39 +21,20 @@ matrix:
...
-
-    # Test spark module for 1.1.1
-    - jdk: "oraclejdk7"
-      env: SPARK_VER="1.1.1" HADOOP_VER="2.3" PROFILE="-Pspark-1.1 -Phadoop-2.3 -Ppyspark" BUILD_FLAG="package -DskipTests" TEST_FLAG="verify" TEST_PROJECTS="-pl zeppelin-interpreter,zeppelin-zengine,zeppelin-server,zeppelin-display,spark-dependencies,spark -Dtest=org.apache.zeppelin.rest.*Test,org.apache.zeppelin.spark* -DfailIfNoTests=false"
-
 before_install:
   - "export DISPLAY=:99.0"
   - "sh -e /etc/init.d/xvfb start"
 
 install:
```
Comment (prabhjyotsingh) [inline .travis.yml:28]: asdfadfasdf

**Comment by dipayanb** [comment]
asdadasd

**Comment by dipayanb** [comment]
dagskjdg

**Pull request closed without merging**